js/extension.js

```diff
@@ -16,7 +16,7 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 const storage = require("./storage.js")
-const hdkey = require('ethereumjs-wallet/hdkey')
+const hdkey = require('ethereumjs-wallet/dist/hdkey')
```

Comment:
Why is this change needed?

Reply:
Required to start debug. It broke gulp and prevents starting

Reply:
All minification is done by webpack now so I think it's better to import unminified(w/o dist) version.

Reply:
Reverted.